x-pack/plugins/security_solution/public/resolver/view/map.tsx

```diff
@@ -10,6 +10,7 @@
 
 import React, { useContext } from 'react';
 import { useSelector } from 'react-redux';
+import { useEffectOnce } from 'react-use';
```

Comment:
:information_source: TIL

Reply:
I wouldn't recommend using this. Just use `useEffect(() => {}, [])`.